src/controller/discoverymgr/mnedc/tunmgr/tun.go

```diff
@@ -19,11 +19,12 @@ package tunmgr
 
 import (
 	"fmt"
+	"github.com/lf-edge/edge-home-orchestration-go/src/common/logmgr"
```

Comment:
Don't move this line of code, it might be conflict with #272

Reply:
Ok.